fem/gridfunc.hpp

```diff
@@ -500,49 +500,120 @@ class GridFunction : public Vector
    virtual void ProjectBdrCoefficientTangent(VectorCoefficient &vcoeff,
                                              const Array<int> &bdr_attr);
 
+   /// @brief Returns ||exsol - u_h||_L2 for vector H1 or L2 elements
```

Comment:
Not necessarily vector. I would drop that word.

Reply:
It could be argued that $vdim=1$ produces a one dimensional vector so this comment would be correct.

Reply:
And it could be argued that people call that case scalar 😄 . But ok, I missed the special one without an array of coefficients. Maybe "vector" in parenthesis?

Reply:
`scalar or vector`?